build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.15
```

Comment:
These changes seem unrelated. Where are they from?

Reply:
I am sorry for this one, I just follow the step from the issue so copy the Dockerfile but I forget to delete that, I fill fix that